op-chain-ops/genesis/migration/params.go

```diff
@@ -0,0 +1,51 @@
+package migration
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// Params contains the configuration parameters used for verifying
+// the integrity of the migration.
+type Params struct {
+	// ExpectedSupplyDelta is the expected delta between the total supply of OVM ETH,
+	// and ETH we were able to migrate. This is used to account for supply bugs in
+	//previous regenesis events.
+	ExpectedSupplyDelta *big.Int
+
+	// IgnoredWithdrawalSlots is a map of storage slots to ignore while validation withdrawal
+	// witness data. The witness data generation script does not take into account reverts,
+	// so we need to ignore certain storage slots that don't exist in the state as a result
+	// of a revert.
+	IgnoredWithdrawalSlots map[common.Hash]bool
+}
+
+var ParamsByChainID = map[int]*Params{
+	1: {
+		// Regenesis 4 (Nov 11 2021) contained a supply bug such that the total OVM ETH
+		// supply was 1.628470012 ETH greater than the sum balance of every account migrated
+		// / during the regenesis. A further 0.0012 ETH was incorrectly not removed from the
+		// total supply by accidental invocations of the Saurik bug (https://www.saurik.com/optimism.html).
+		new(big.Int).SetUint64(1627270011999999992),
+
+		map[common.Hash]bool{},
+	},
+	5: {
+		new(big.Int),
+
+		// The below reverted messages are current as of 12-21-2022. We may encounter more of these as
```

Comment:
We probably want to update the witness generation scripts to handle this

Reply:
This is removed now as per our discussion with Kelvin this morning.